packages/@uppy/dashboard/src/utils/getFileTypeIcon.js

```diff
@@ -39,6 +46,19 @@ module.exports = function getIconByMime (fileType) {
     }
   }
 
+  if (fileTypeGeneral === 'application' && (
+    fileTypeSpecific === 'zip' ||
+    fileTypeSpecific === 'x-7z-compressed' ||
+    fileTypeSpecific === 'x-rar-compressed' ||
+    fileTypeSpecific === 'x-gtar' ||
+    fileTypeSpecific === 'x-apple-diskimage')
```

Comment:
@goto-bus-stop do you maybe have better ideas on detecting archives? I could move the `fileTypeSpecific`s to an array, but other than that :)